app/services/workflow_link_service.rb

```diff
@@ -1,25 +1,33 @@
+require 'remote_tagging_service'
 class WorkflowLinkService
   attr_accessor :workflow_id
 
-  TAG_PREFIX = "/approval/workflows/".freeze
+  TAG_NAMESPACE = 'approval'.freeze
+  TAG_NAME      = 'workflows'.freeze
 
   def initialize(workflow_id)
     self.workflow_id = workflow_id
   end
 
   def link(tag_attrs)
     TagLink.find_or_create_by!(tag_link(tag_attrs))
+    RemoteTaggingService.new(tag_attrs).process('add', approval_tag)
```

Comment:
What happen if the tag already exists at the remote service? Will it return an error? Should we consume the error and continue?

Reply:
@bzwei If it exists we get back a 200 and nothing bad happens